Assignment 2.Rmd

```diff
@@ -87,12 +94,53 @@ plot(g,layout=layout.fruchterman.reingold, vertex.color=VERTEX$gender, edge.widt
 
 In Part II your task is to [look up](http://igraph.org/r/) in the igraph documentation and create a graph that sizes the student vertices in terms of the number of disciplinary actions they have recieved, and the teachers in terms of the number of disciplinary actions they have given out.
 
+```{r}
+EDGE.st <- EDGE %>% group_by(to) %>% summarize(sum(count))
+names(EDGE.st) <- c("id", "count")
+EDGE.tc <- EDGE %>% group_by(from) %>% summarise(sum(count))
+EDGE.tc
+names(EDGE.tc) <- c("id", "count")
+total <- bind_rows(EDGE.st, EDGE.tc)
```

Comment:
Be careful you are binding variables of the same type (EG - `factor` to `factor)